--- a/src/views/ComputerManagement/ComputerGroupManagement/ComputerGroupManagement.vue
+++ b/src/views/ComputerManagement/ComputerGroupManagement/ComputerGroupManagement.vue
@@ -61,6 +61,7 @@
                         @click="setSelectedPluginTab('security-management')"
                         :label="$t('computer.plugins.button.security')"
                         v-if="isExistPrivilege([
+                            'ROLE_NETWORK_MANAGER',
                             'ROLE_DEVICE_MANAGEMENT',
                             'ROLE_USB_RULE'
                         ])"
